modules/oe_bootstrap_theme_helper/tests/src/Kernel/BclIconPathOverrideTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme_helper\Kernel;
+
+use Drupal\Core\Theme\ThemeManagerInterface;
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests the bcl_icon_path theme override functionality.
+ */
+class BclIconPathOverrideTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'system',
+    'theme_test',
+    'oe_bootstrap_theme_helper',
```

Comment:
Not needed helper module, the theme does the preprocess.
Module theme_test can be removed too.

Reply:
done.